buildSrc/src/main/groovy/com/google/android/perftesting/PerfTestTaskGeneratorPlugin.groovy

```diff
@@ -119,13 +119,17 @@ public class PerfTestTaskGeneratorPlugin implements Plugin<Project> {
                         outputLine = outputLine.trim()
                         if (!outputLine.startsWith("List of devices attached") && !"".equals(outputLine)) {
                             String[] lineParts = outputLine.split(/\s+/) // The regex groups whitespace.
-                            if (lineParts.length != 6) {
+                            if (lineParts.length > 6 || lineParts.length < 5 ) {
+                                println(lineParts.length)
```

Comment:
Temporary debugging code (`println`) should be removed. Use logging instead and make it more descriptive if the debug information (the number of parts) is still helpful to future developers.